lib/ArrayInterfaceCUDA/Project.toml

```diff
@@ -8,6 +8,7 @@ ArrayInterface = "4fba245c-0d91-5ea0-9b3e-6abc04ee57a9"
 ArrayInterfaceGPUArrays = "6ba088a2-8465-4c0a-af30-387133b534db"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
This doesn't need to be a dependency, just a test dependency

Reply:
I made some changes but am unsure whether I did it correctly.